desktop/storeService.ts

```diff
@@ -1,11 +1,26 @@
-const Store = require('electron-store');
+import { nativeTheme } from 'electron';
+import Store from 'electron-store';
+import { TypedSchemaStore } from '../shared/types';
 
 class StoreService {
-  static store: typeof Store;
+  public static store: Store<TypedSchemaStore>;
 
-  static init() {
+  constructor() {
```

Comment:
No-no-no! We're using it as a Singletone, so there shouldn't be a public constructor. It should be private or absent.

Reply:
Okay

Reply:
Fixed